.github/install_dependencies.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+echo "Hello this is first attempt to run bash script on github actions"
+OS=$(cat /etc/*release | grep ^NAME | tr -d 'NAME="') 
+ID=$(cat /etc/*release | grep ^ID | tr -d 'ID="') 
+VERSION_ID=$(cat /etc/*release | grep ^VERSION_ID | tr -d 'VERSION_ID="') 
+
+echo $OS
+echo $ID
+echo $VERSION_ID
+
+if [ $VERSION_ID = "8" ]
```

Comment:
Can't we use linchpin setup for this?

Reply:
@14rcole I think Linchpin setup is available after installation ( it is useful while installing beaker, libvirt or any specific provider dependencies, unlike prereqs )
Further, the epel release packages and git are to be installed before linchpin is installed. 
since we are not using docker files here, I have written this bash script to install prereqs in the container.